test-case/manual/check-simple-probe-capture.md

```diff
@@ -0,0 +1,86 @@
+# Probes test
+Simple probes functionality test.
+
+# Preconditions
+1. Linux kernel should have these config options enabled:
+   CONFIG_SND_SOC_SOF_DEBUG_PROBES=y
+   CONFIG_SND_SOC_SOF_HDA_PROBES=y
+
+   These options are not user selectable and can not be enabled with
+   'make menuconfig'. Edit .config manually instead.
```

Comment:
No, you can not edit the .config, these options are config selected:
SND_SOC_SOF_INTEL_SKL/APL/CNL/ICL/TGL/MTL selects SND_SOC_SOF_HDA_COMMON, which selects SND_SOC_SOF_HDA_PROBES, which SND_SOC_SOF_DEBUG_PROBES.
The probes client is only available for HDA platforms.

Reply:
I don't know about these CONFIG_ in particular but in general `.config` should never be edited manually. Any "real world" Kconfig system has complex dependency rules and manual edition is likely to break consistency.

https://docs.zephyrproject.org/2.6.0/guides/kconfig/setting.html#visible-and-invisible-kconfig-symbols
> An invisible symbol is a symbol without a prompt. Invisible symbols are not shown in the interactive configuration interfaces, and users have no direct control over their value. They instead get their value from defaults or from other symbols.

If some hack is really required locally and temporarily, it should be a hack in a Kconfig file, not in the .config file.

(yes I know it's the kernel but `kconfig.py` is bug-for-bug compatible and has much better documentation. It's what we use in SOF too)

I see this has been removed in the current version.